src/grammar/src/visitor/ruleCondition.visitor.ts

```diff
@@ -14,10 +14,10 @@ export class RuleConditionVisitor extends ParseTreeVisitor {
         break;
       }
       case RulesParser.RULE_filterExpression: {
-        if (ctx.children?.[0].text) {
-          this.condition.objectType = RuleObjectTypes[ctx.children?.[0].text];
+        if (ctx.children?.[0]?.text) {
+          this.condition.objectType = RuleObjectTypes[ctx.children?.[0]?.text];
```

Comment:
redundant optional chaining operators in this line

Reply:
fixed